cypress/components/numeral-date/numeral-date.cy.tsx

```diff
@@ -440,16 +440,6 @@ context("Tests for NumeralDate component", () => {
         CHARACTERS.STANDARD
       );
     });
-
-    it("should have the expected border radius styling when no search button enabled", () => {
```

Comment:
**question(non-blocking):** I can see you've removed this test (we'll just just gloss over the reference to a different component altogether) but do we still have a test to check the `border-radius` on this component?

Reply:
There is no other reference to the search button so I think this was added by mistake to the description of the test. 

I'm also going to completely remove setting the `border-radius` from the style file as it is the same one as `Textbox`.